--- mysite/mysite/settings.py
+++ mysite/mysite/settings.py
@@ -32,9 +32,14 @@
 
 
 # Application definition
+AUTHENTICATION_BACKENDS = [
+    'django.contrib.auth.backends.ModelBackend',
+    'djangocms_oidc.auth.DjangocmsOIDCAuthenticationBackend',
+]
 
-
-
+# OIDC classes
+OIDC_AUTHENTICATE_CLASS = "djangocms_oidc.views.DjangocmsOIDCAuthenticationRequestView"
+OIDC_CALLBACK_CLASS = "djangocms_oidc.views.DjangocmsOIDCAuthenticationCallbackView"
 
 
 ROOT_URLCONF = 'mysite.urls'
@@ -136,7 +141,8 @@
     'cms.middleware.user.CurrentUserMiddleware',
     'cms.middleware.page.CurrentPageMiddleware',
     'cms.middleware.toolbar.ToolbarMiddleware',
-    'cms.middleware.language.LanguageCookieMiddleware'
+    'cms.middleware.language.LanguageCookieMiddleware',
+    'djangocms_oidc.middleware.OIDCSessionRefresh',
 ]
 
 INSTALLED_APPS = [
@@ -179,6 +185,12 @@
     'djangocms_snippet',
     'djangocms_googlemap',
     'djangocms_video',
+    'multiselectfield',
+    'django_countries',
+    'mozilla_django_oidc',
+    'djangocms_oidc',
+    'aldryn_forms',
+    'djangocms_oidc_form_fields',
     'mysite'
 ]
 
--- mysite/mysite/urls.py
+++ mysite/mysite/urls.py
@@ -15,6 +15,8 @@
 urlpatterns = [
     url(r'^sitemap\.xml$', sitemap,
         {'sitemaps': {'cmspages': CMSSitemap}}),
+    url(r'^oidc/', include('mozilla_django_oidc.urls')),
+    url(r'^djangocms-oidc/', include('djangocms_oidc.urls')),
 ]
 
 urlpatterns += i18n_patterns(
